<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectProductDialog</class>
 <widget class="QDialog" name="SelectProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>823</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>gLabels - Select Product</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="modeNotebook">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="search">
          <attribute name="title">
           <string>Search all</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="topMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <item>
                <widget class="QLineEdit" name="searchEntry">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>220</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="placeholderText">
                  <string>Search</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Filter by paper size</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <item>
                   <widget class="QCheckBox" name="pageSizeIsoCheck">
                    <property name="text">
                     <string>ISO sizes</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="pageSizeUsCheck">
                    <property name="text">
                     <string>US sizes</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="pageSizeOtherCheck">
                    <property name="text">
                     <string>Other</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Filter by category</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QRadioButton" name="allCategoriesRadio">
                  <property name="text">
                   <string>All</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="selectedCategoriesRadio">
                  <property name="text">
                   <string>Selected</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="categoriesCheckContainer" native="true">
                  <layout class="QGridLayout" name="gridLayout">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item row="0" column="0">
                    <layout class="QVBoxLayout" name="categoriesLayout">
                     <property name="spacing">
                      <number>3</number>
                     </property>
                     <property name="leftMargin">
                      <number>20</number>
                     </property>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>3</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Search entire product database.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="recent">
          <attribute name="title">
           <string>Recent</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Select from recently used products.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="glabels::TemplatePicker" name="templatePicker">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glabels::TemplatePicker</class>
   <extends>QListWidget</extends>
   <header>TemplatePicker.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pageSizeUsCheck</sender>
   <signal>clicked()</signal>
   <receiver>SelectProductDialog</receiver>
   <slot>onPageSizeCheckClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>templatePicker</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>SelectProductDialog</receiver>
   <slot>onTemplatePickerSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>660</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageSizeIsoCheck</sender>
   <signal>clicked()</signal>
   <receiver>SelectProductDialog</receiver>
   <slot>onPageSizeCheckClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectProductDialog</receiver>
   <slot>onCancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>955</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>660</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchEntry</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SelectProductDialog</receiver>
   <slot>onSearchEntryTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allCategoriesRadio</sender>
   <signal>clicked()</signal>
   <receiver>SelectProductDialog</receiver>
   <slot>onCategoryRadioClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectedCategoriesRadio</sender>
   <signal>clicked()</signal>
   <receiver>SelectProductDialog</receiver>
   <slot>onCategoryRadioClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modeNotebook</sender>
   <signal>currentChanged(int)</signal>
   <receiver>SelectProductDialog</receiver>
   <slot>onModeTabChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSearchClearButtonClicked()</slot>
  <slot>onTemplatePickerSelectionChanged()</slot>
  <slot>onOrientationRadioToggled()</slot>
  <slot>onSearchEntryTextChanged()</slot>
  <slot>onCancelButtonClicked()</slot>
  <slot>onSelectButtonClicked()</slot>
  <slot>onPageSizeCheckClicked()</slot>
  <slot>onCategoryRadioClicked()</slot>
  <slot>onModeTabChanged()</slot>
 </slots>
</ui>
